src/qibolab/instruments/dummy_oscillator.py

```diff
@@ -0,0 +1,31 @@
+from qibolab.instruments.abstract import LocalOscillator
+
+
+class DummyLocalOscillator(LocalOscillator):
+    """Dummy local oscillator driver.
+
+    Useful for using with the Quantum Machines simulator.
+    """
+
+    def __init__(self, name, address):
+        super().__init__(name, address)
+        self.power: float
+        self.frequency: float
```

Comment:
We can convert to private.

Reply:
We can, but then we would have to add getters and setters, as these are used outside the class. This will be fixed as part of the general LO refactoring for #297.